puppet/environments/box/modules/profiles/files/box/cli/scripts/Command/Share.php

```diff
@@ -61,9 +69,32 @@ protected function _check()
         }
     }
 
+    protected function _removeVhostOverride()
+    {
+        $site_enabled = '/etc/apache2/sites-enabled/2-ngrok.conf';
+        $site_available = '/etc/apache2/sites-available/2-ngrok.conf';
+        $restart = false;
+
+        if (file_exists($site_enabled))
+        {
+            `sudo rm /etc/apache2/sites-enabled/2-ngrok.conf`;
+            $restart = true;
+        }
+
+        if (file_exists($site_available))
+        {
+            `sudo rm /etc/apache2/sites-available/2-ngrok.conf`;
+            $restart = true;
+        }
+
+        if ($restart){
+            `sudo /etc/init.d/apache2 restart > /dev/null 2>&1`;
```

Comment:
@yiendos We should be managing this process through systemd now, so this command should look like `sudo systemctl restart apache2`